src/TreeProgram.java

```diff
@@ -0,0 +1,21 @@
+
+public class TreeProgram {
+	
+	// removed instance variables.
+	void findVisibleTree(int[] arr){
+		int max=arr[0], count=1; // updated variable name as max.
+		for (int index= 0; index < arr.length; index++) {
```

Comment:
Index should start from 1 as we already define max variable value from 0 so, we compare value from index 1
you know the changes already

Reply:
Thanks Harsh, yes I got this point and completely agree on this.